### DISCUSSION TITLE
Config files not working on version 8.11.12.6

### DISCUSSION THREAD

**Discussion by Ferdi-SQ** [d2phap/ImageGlass#1819] [General]
Hi, according to the documentation here: [https://imageglass.org/docs/app-configs](https://imageglass.org/docs/app-configs) there should be a way to setup configuration parameters on the software itself.

However, when I install the application and change the JSON config file, it does not pick up on these settings. It does not matter whether or not I use a igconfig.json, igconfig.default.json or igconfig.admin.json.

Is there a way to change the default application behaviour within the registry of the local machine? Or some other way for me to prevent the application from looking for updates?

Kind regards.

**Comment by d2phap** [comment]
The docs was updated for version 9. In v8, IG uses XML files instead. Similar to v9:
- `igconfig.default.xml` -> `igconfig.default.json`
- `igconfig.xml` -> `igconfig.json`
- `igconfig.admin.xml` -> `igconfig.admin.json`

For the content of the XML file, you can copy the content of `igconfig.xml` file located in `%AppData%\ImageGlass\` folder.

Also note that, IG v8 DOES NOT support passing settings as command-line arguments. Each command must begin with a `/`.

**Reply by d2phap** [reply]
Use this link for v8 docs: https://web.archive.org/web/20230825004513/https://imageglass.org/docs/app-configs

**Reply by Ferdi-SQ** [reply]
Hi, thanks for your information. Is it possible to link to the old documentation on the page of version 8?
